package.json

```diff
@@ -31,7 +31,7 @@
     "test": "mt tests --quiet"
   },
   "dependencies": {
-    "glob": "^10.2.2",
+    "glob": "~10.1.0",
```

Comment:
A temporary fix so that `lint` does not error:

```sh
❯ yarn lint
yarn run v1.22.19
$ eslint . --cache
There was a problem loading formatter: /ember-codemod-pod-to-octane/node_modules/eslint/lib/cli-engine/formatters/stylish
Error: require() of ES Module /ember-codemod-pod-to-octane/node_modules/strip-ansi/index.js from /ember-codemod-pod-to-octane/node_modules/eslint/lib/cli-engine/formatters/stylish.js not supported.
Instead change the require of index.js in /ember-codemod-pod-to-octane/node_modules/eslint/lib/cli-engine/formatters/stylish.js to a dynamic import() which is available in all CommonJS modules.
```

- https://github.com/ijlee2/ember-codemod-pod-to-octane/actions/runs/4869174164/jobs/8683379549
- https://github.com/yargs/cliui/compare/master...isaacs:cliui:main